pinecone/grpc/index_grpc.py

```diff
@@ -15,8 +15,8 @@
 
```

Comment:
This will be good for actually adding some documentation around the GRPC bits, thank you. 👍